src/Presentation/HtmlPredicatesPresenter.php

```diff
@@ -10,36 +10,74 @@
 
 class HtmlPredicatesPresenter implements PredicatesPresenter {
 
+	private const ASTERISK = '<code>*</code>';
+
 	private string $response = '';
-	private PredicateList $predicates;
+	private PredicateList $localSettingsPredicates;
+	private PredicateList $wikiPredicates;
 
-	public function presentPredicates( PredicateList $predicates ): void {
-		$this->predicates = $predicates;
-		$this->response = $this->createIntro() . $this->createList();
+	public function presentPredicates( PredicateList $localSettingsPredicates, PredicateList $wikiPredicates ): void {
+		$this->localSettingsPredicates = $localSettingsPredicates;
+		$this->wikiPredicates = $wikiPredicates;
+		$this->response = $this->createIntro() . $this->createList() . $this->createFooter();
 	}
 
 	private function createIntro(): string {
 		return Html::rawElement( 'p', [], wfMessage( 'wikibase-rdf-config-list-intro' )->text() );
 	}
 
 	private function createList(): string {
-		$predicates = $this->predicates->asArray();
+		$localSettingsPredicates = $this->localSettingsPredicates->asArray();
+		$wikiPredicates = $this->wikiPredicates->asArray();
+		$count = count( $localSettingsPredicates ) + count ( $wikiPredicates );
```

Comment:
Oh no, too much spaaaaaaace (

Reply:
I keep running only `make test` before pushing. I guess I need to permanently change my local makefile to have that run `ci` instead.